ocs_ci/framework/pytest_customization/ocscilib.py

```
@@ -1132,12 +1132,66 @@ def set_stop_flags_for_all_clusters(stop_requested, graceful_stop):
         ocsci_config.RUN["graceful_stop"] = graceful_stop
 
 
+def _check_disabled_components(item):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_check_disabled_components`

Add return type annotation: `None`

(ANN202)